packages/openapi/src/openapi-request-builder.ts

```diff
@@ -24,14 +24,15 @@ import {
   OriginOptions,
   HttpMiddleware
 } from '@sap-cloud-sdk/http-client/internal';
+import { CustomRequestConfig } from '@sap-cloud-sdk/http-client/src/http-client-types';
```

Comment:
[req] This should be imported from the toplevel (and if not possible, also exported).
Otherwise this will fail for users, because they don't have a `src` directory.

```suggestion
import { CustomRequestConfig } from '@sap-cloud-sdk/http-client';
```

Reply:
For it to be imported from the top level, you will also need to include the type in the index.ts 👀